src/index.ts

```diff
@@ -1,340 +1,173 @@
-import { dirname, relative, resolve, extname } from "path";
-import ts from "typescript";
-import { parse } from "url";
-import { existsSync } from "fs";
+// noinspection ES6UnusedImports
+import {} from 'ts-expose-internals'
```

Comment:
Because TS is weird 😆 I'm actually hoping I can find a better solution (I wrote the library), but so far, that's the best I can come up with.

The options we have are to add it to `typeRoots` in tsconfig or to import it in one of the project files. 

Unfortunately, with `import 'ts-expose-internals`, that line gets added to the output and thus becomes a broken import. We use the empty brackets so TS recognizes it as type-only. They made that the standard behaviour since 3.8, I believe.

The other route is to add it to `typeRoots`, but I try not to do that, because I believe it overrides the automatic `typeRoots` setting. We usually don't want that, though it would likely be fine in this case. Probably better to just stick with the prescribed way.